lsp-methods.el

```diff
@@ -1722,6 +1754,7 @@ command COMMAND and optionsl ARGS"
 ;; (defalias 'lsp-on-change #'lsp--text-document-did-change)
 (defalias 'lsp-completion-at-point #'lsp--get-completions)
 (defalias 'lsp-error-explainer #'lsp--error-explainer)
+(defalias 'lsp-restart-workspace #'lsp--restart-workspace)
```

Comment:
Why this alias? If `lsp--restart-workspace` is meant to be called by users, just make it public.

Reply:
I am just cargo culting the code, elisp is not my primary language :)